From: Jan Engelhardt <ej@inai.de>
Date: 2025-11-17 20:53:47.057848785 +0100

SDL-devel depends on glu and pulls the headers in, but not so
for sdl_compat. Address the compile failures ensuring from that.
---
 BlockOut/GLApp/GLApp.h    |    1 +
 BlockOut/GLApp/GLFont.h   |    1 +
 BlockOut/GLApp/GLMatrix.h |    1 +
 BlockOut/GLApp/GLSprite.h |    1 +
 4 files changed, 4 insertions(+)

Index: BL_SRC/BlockOut/GLApp/GLApp.h
===================================================================
--- BL_SRC.orig/BlockOut/GLApp/GLApp.h
+++ BL_SRC/BlockOut/GLApp/GLApp.h
@@ -4,6 +4,7 @@
 // -------------------------------------------
 #include <SDL.h>
 #include <SDL_opengl.h>
+#include <GL/glu.h>
 #include <string.h>
 #include <stdlib.h>
 
Index: BL_SRC/BlockOut/GLApp/GLFont.h
===================================================================
--- BL_SRC.orig/BlockOut/GLApp/GLFont.h
+++ BL_SRC/BlockOut/GLApp/GLFont.h
@@ -2,6 +2,7 @@
 // Simple 2D font
 // -----------------------------------------------
 #include "SDL_opengl.h"
+#include <GL/glu.h>
 
 #ifndef _GLFONT2DH_
 #define _GLFONT2DH_
Index: BL_SRC/BlockOut/GLApp/GLMatrix.h
===================================================================
--- BL_SRC.orig/BlockOut/GLApp/GLMatrix.h
+++ BL_SRC/BlockOut/GLApp/GLMatrix.h
@@ -2,6 +2,7 @@
 // 3x3 Matrix class
 // -------------------------------------
 #include <SDL_opengl.h>
+#include <GL/glu.h>
 
 #ifndef _GLMATRIXH_
 #define _GLMATRIXH_
Index: BL_SRC/BlockOut/GLApp/GLSprite.h
===================================================================
--- BL_SRC.orig/BlockOut/GLApp/GLSprite.h
+++ BL_SRC/BlockOut/GLApp/GLSprite.h
@@ -2,6 +2,7 @@
 // 2D sprites
 // -----------------------------------------------
 #include "SDL_opengl.h"
+#include <GL/glu.h>
 
 #ifndef _SPRITE2DH_
 #define _SPRITE2DH_
